### DISCUSSION TITLE
v0.11

### DISCUSSION THREAD

**Discussion by anitagraser** [anitagraser/movingpandas#237] [Announcements]
Published at https://anaconda.org/conda-forge/movingpandas

New features:

- #37 New minimum and Hausdorff distance measures
- #233 New functions to add a timedelta column and get the trajectory sampling interval

Includes fixes and enhancements for:

- #231 Return more representative stop point
- #230 Skip missing optional deps on import
- [Faster creation of TrajectoryCollection objects](https://github.com/anitagraser/movingpandas/commit/ae892242c62dcf87488dcb67c81a0161569c7039)
- [Faster trajectory length calculation](https://github.com/anitagraser/movingpandas/commit/24a193cf7cd253abd918ec92fd21bc1c62edb6b9)
- [Fixed CRS handling for GeoPandas 0.11.0](https://github.com/anitagraser/movingpandas/commit/f44f0bcf41477b594d9735c8c6aa77958871c987)

Behind the scenes:

- [Updated papers using MovingPandas](https://github.com/anitagraser/movingpandas/commit/81290d820d668938113bd3c0d93b9a11e2fa344d)

<hr /><em>This discussion was created from the release <a href='https://github.com/anitagraser/movingpandas/releases/tag/v0.11'>v0.11</a>.</em>

**Comment by luismanriqueruiz** [comment]
Wonderful, keep it up!
